packager/packager.cc

```diff
@@ -402,6 +402,12 @@ bool StreamInfoToTextMediaInfo(const StreamDescriptor& stream_descriptor,
     text_media_info->set_bandwidth(kDefaultTextBandwidth);
   }
 
+  if (!stream_descriptor.dash_roles.empty()) {
+    for (auto dash_role : stream_descriptor.dash_roles) {
```

Comment:
@sr1990 Thanks for addressing this bug! Have you verified that the change works? Can you update the end to end test? You may add a text stream with role to this test: https://github.com/google/shaka-packager/blob/master/packager/app/test/packager_test.py#L671.

Use constant reference to avoid copying:

```c++
for (const auto& dash_role : ...)
```

Reply:
@kqyang Thanks for reviewing. Yes I have verified that the change works. Have added text stream to the mentioned test in packager_test.py.

Reply:
@sr1990 Cool. Thanks. I have merged the change.